src/quantum_system_templates/rydberg.jl

```diff
@@ -69,6 +73,8 @@ function RydbergChainSystem(;
     local_detune::Bool=false,
     all2all::Bool=true,
     ignore_Y_drive::Bool=false,
+    T_max::Float64=10.0,
+    drive_bounds::Vector{<:Union{Tuple{Float64, Float64}, Float64}}=ignore_Y_drive ? [1.0, 1.0] : [1.0, 1.0, 1.0],
```

Comment:
I'm seeing this drive_bounds vec used differently in a few places. I thought it was
[(a_1_low, a_1_high), (a_2_low, a_2_high)]. I think its to our benefit to choose one of these, or document the many ways it can be expressed.